ui/src/app/utils/index.tsx

```diff
@@ -628,3 +628,20 @@ export function validateInputForMySQL(searchString: string): Array<string> {
   }
   return Array.from(inputErrors);
 }
+
```

Comment:
Lenses allow you to "focus" on specific data.
This is a specific implementation of a lens that allows us to normalize property names that represent the same values. This allows us to write one function to operate on those values regardless of how the data was keyed.

Ramda has a repl on the page if you want to play around with lenses.